zmon_slr/main.py

```diff
@@ -1,26 +1,34 @@
 #!/usr/bin/env python3
 
-import os
 import json
 import numbers
+import os
 import subprocess
 
 import click
 import requests
 import zign.api
-
-from clickclick import AliasedGroup, Action, error, warning, fatal_error
+from clickclick import Action, AliasedGroup, error, fatal_error, warning
 from zmon_cli.client import Zmon
 
 from zmon_slr.client import Client, SLRClientError
 from zmon_slr.generate_slr import generate_weekly_report
 
-
 DEFAULT_CONFIG_FILE = '~/.zmon-slr.yaml'
 CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
 SLR_TOKEN = os.environ.get('SLR_TOKEN')
 
 AGG_TYPES = ('average', 'weighted', 'sum', 'min', 'max', 'minimum', 'maximum')
+LIGHTSTEP_METRICS = (
```

Comment:
could `LIGHTSTEP_METRICS` be defined in Lightstep source file? and i guess `AGG_TYPES` in zmon source? and `validate_*_source` in corresponding sources?

Reply:
talked in person. this is not possible as zmon slr cannot import from backend files. This validation should ultimately be deleted since backend already does validation, but this can be done as a separate PR